web/test/pages/profile.test.tsx

```diff
@@ -184,7 +184,7 @@ describe("profile", () => {
     clickSave();
     await waitFor(() => {
       expect(
-        subject.queryByText(templateEval(OnboardingDefaults.errorTextMinimumNumericField, { length: "10" }))
+        subject.queryByText(templateEval(OnboardingDefaults.errorTextMinimumNumericField, { length: "9" }))
```

Comment:
why did this change?

Reply:
How it calculated the length in reference to the hyphen within the field changed, this is more accurate.